tests/utils.js

```diff
@@ -520,7 +520,6 @@ module.exports = {
     captureOnlyFailedSpecs: true,
     reportOnlyFailedSpecs: false,
     showQuickLinks: true,
-    dest: `tests/results/`,
```

Comment:
Is this intentional? It will break the artifact path when run on github.  If it is, then the build.yml needs updated as well.